Core/Specifications/TripSpecification.cs

```diff
@@ -46,5 +46,19 @@ public GetUnactiveById(int tripId)
                 Query.Where(t => t.Id == tripId && (!t.IsActive && !t.IsEnded));
             }
         }
+
+        internal class GetByCreatorId : Specification<Trip>
```

Comment:
This class specified for routes, so rename it as GetRoutesByCreatorId for instance